maps/WIP/permafrost.dmm

```diff
@@ -487,17 +487,50 @@
+"jP" = (/obj/structure/table/modern/table,/obj/item/clothing/head/ghillie/winter,/obj/item/clothing/head/ghillie/winter,/obj/item/clothing/head/ghillie/winter,/obj/item/clothing/suit/storage/ghillie/winter,/obj/item/clothing/suit/storage/ghillie/winter,/obj/item/clothing/suit/storage/ghillie/winter,/turf/floor/plating/concrete,/area/caribbean/no_mans_land/capturable/one)
+"jQ" = (/obj/structure/table/modern/table,/obj/item/clothing/head/ww2/sov_ushanka,/obj/item/clothing/head/ww2/sov_ushanka,/obj/item/clothing/head/ww2/sov_ushanka,/obj/item/clothing/head/ww2/sov_ushanka,/obj/item/clothing/head/ww2/sov_ushanka,/obj/item/clothing/head/ww2/sov_ushanka,/turf/floor/plating/concrete,/area/caribbean/no_mans_land/capturable/one)
+"jR" = (/obj/structure/closet/anchored,/obj/item/clothing/head/helmet/modern/zsh1,/obj/item/clothing/head/helmet/modern/zsh1,/obj/item/clothing/head/helmet/modern/zsh1,/obj/item/clothing/head/helmet/modern/zsh1,/obj/item/clothing/head/helmet/modern/zsh1,/obj/item/clothing/head/helmet/modern/zsh1,/turf/floor/plating/concrete,/area/caribbean/no_mans_land/capturable/one)
+"jS" = (/obj/structure/closet/anchored,/obj/item/violin,/turf/floor/carpet/pinkcarpet,/area/caribbean/void/caves)
+"jT" = (/obj/structure/table/modern/table,/obj/item/violin,/turf/floor/plating/steel,/area/caribbean/void/caves)
+"jU" = (/obj/item/weapon/material/hatchet/steel,/obj/item/weapon/material/hatchet/steel,/obj/item/weapon/material/hatchet/steel,/obj/item/weapon/material/hatchet/steel,/obj/item/weapon/material/hatchet/steel,/obj/item/weapon/material/hatchet/steel,/obj/item/weapon/material/hatchet/battleaxe/steel,/obj/item/weapon/material/hatchet/battleaxe/steel,/obj/item/weapon/material/hatchet/battleaxe/steel,/obj/item/weapon/material/hatchet/battleaxe/steel,/obj/structure/closet/crate/empty,/turf/floor/plating/concrete,/area/caribbean/no_mans_land/capturable/one)
+"jV" = (/obj/item/weapon/material/machete1,/obj/item/weapon/material/machete1,/obj/item/weapon/material/machete1,/obj/item/weapon/material/machete1,/obj/item/weapon/material/machete1,/obj/item/weapon/material/machete,/obj/item/weapon/material/machete,/obj/item/weapon/material/machete,/obj/item/weapon/material/machete,/obj/item/weapon/material/machete,/obj/item/weapon/material/twohanded/fireaxe,/obj/item/weapon/material/twohanded/fireaxe,/obj/item/weapon/material/twohanded/fireaxe,/obj/item/weapon/material/twohanded/fireaxe,/obj/item/weapon/material/twohanded/fireaxe,/obj/item/weapon/material/twohanded/fireaxe,/obj/item/weapon/material/twohanded/fireaxe,/obj/structure/closet/crate/empty,/turf/floor/plating/concrete,/area/caribbean/no_mans_land/capturable/one)
+"jW" = (/obj/structure/lamp/lamp_big/alwayson{dir = 1; icon_state = "tube"},/turf/floor/plating/concrete,/area/caribbean/no_mans_land/capturable/one)
+"jX" = (/obj/structure/closet/anchored,/obj/item/clothing/accessory/catears,/turf/floor/carpet/pinkcarpet,/area/caribbean/void/caves)
```

Comment:
`/obj/item/clothing/accessory/catears` What Epoch is this map?

Reply:
"do not add in catears into a 2050 modern map"